app/src/main/resources/conf-apiauthkeys.properties

```diff
@@ -1,3 +1,3 @@
 #API Auth Keys
-apiauth.keys[0]=9d3868d1-ec15-4889-8002-2bff1b50ba62
-apiauth.keys[1]=ec4624eb-a02d-4d20-bac6-095b98a792a2
+apiauth.keys[0]=${APIAUTHKEYS-01:9d3868d1-ec15-4889-8002-2bff1b50ba62}

```

Comment:
Does this still leave the security information in the public repo?

Reply:
Have APIAUTHKEYS keys put in setenv script in secret repo?

Reply:
I deleted the default values for the apikeys to avoid confusion; the APIkeys will be set as environment variables in the k8 secrets.